src/libs/actions/ReportActions.js

```diff
@@ -111,9 +111,14 @@ function getLastVisibleMessageText(reportID) {
     );
 }
 
+function getLastAction(reportID) {
```

Comment:
JSdocs are required when params exist.

Reply:
I'm also not too big of a fan of this method. I think it would be better to have `OptionsListUtils` connect to Onyx to keep track of the last report action. Action files should be for the UI to trigger API requests and manage where the Onyx data goes. Using Action files for getters is kind of an anti-pattern.

Reply:
Removed that function and added a `reportActions` subscription in `OptionsListUtils` to get the last action of a report.